src/test/java/uk/gov/pay/adminusers/expungeandarchive/service/ExpungeAndArchiveHistoricalDataServiceTest.java

```diff
@@ -244,6 +244,32 @@ void shouldArchiveServiceWithoutTransactionsButCreatedBeforeTheServicesEligibleF
             verify(mockServiceRoleDao).removeUsersFromService(serviceEntity.getId());
         }
 
+        @Test
+        void shouldArchiveServiceWithoutCreatedDateAndFirstButFirstCheckedForArchivalDateIsBeforeTheServicesEligibleForArchivingDate() {
```

Comment:
I'm really not a fan of this method name...it's not particularly understandable/readable... Perhaps separating with underscores might help. I would try to find a more concise method name in any case.